src/plot_api/plot_api.js

```diff
@@ -245,9 +245,7 @@ exports.plot = function(gd, data, layout, config) {
                 .attr('height', fullLayout.height);
         }
 
-        return Lib.syncOrAsync([
-            subroutines.layoutStyles
-        ], gd);
+        return Plots.previousPromises(gd);
```

Comment:
@alexcjohnson tests are :white_check_mark: after removing the extra `layoutStyles` call. Good riddance :boat: